pages/create-issue.tsx

```diff
@@ -1,48 +1,101 @@
 import { GetStaticProps } from 'next'
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import BeproService from '../services/bepro';
 import GithubMicroService from '../services/github-microservice';
+import { setLoadingAttributes } from '../providers/loading-provider';
+import { BeproBalance } from '../helpers/bepro-balance';
+import InputNumber from '../components/inputNumber';
 
-export default function PageCreateIssue() {
+interface Amount {
+  value?: string,
+  formattedValue: string,
+  floatValue?: number
+}
 
+export default function PageCreateIssue() {
 
   const [issueTitle, setIssueTitle] = useState("");
   const [issueDescription, setIssueDescription] = useState("");
-  const [issueAmount, setIssueAmount] = useState<number>(0);
+  const [issueAmount, setIssueAmount] = useState<Amount>({ value: '0', formattedValue: '0', floatValue: 0});
+  const [balance, setBalance] = useState<string>('0');
   const [allowedTransaction, setAllowedTransaction] = useState<boolean>(false);
 
+  const useBeproBalance = async () => {
+    setBalance(await BeproBalance())
+  }
+
+  useEffect(() => {
+    useBeproBalance()
+  },[])
+
   // TODO add loaders since is slow on metamask
   const allow = async (evt) => {
-    evt.preventDefault();
-    await BeproService.login();
-    const beproAddress = await BeproService.getAddress();
-    const payload = {
-      amount: issueAmount,
-      address: beproAddress
-    }
-    const res = await BeproService.network.approveTransactionalERC20Token();
-    const resApproved = await BeproService.network.isApprovedTransactionalToken(payload);
-    if (resApproved) {
+    evt.preventDefault();    
+    try {
```

Comment:
don't use try catch without logging the catch as that will not show on the console log as an error and you will throw future coders off scent if a bug is found here.

In general, avoid using try-catch formula for more than one statement, and if you really have to, then PLEASE log what broken on the catch.

Reply:
Done. -> [991a937, bd13aeb]. does this solve?

Reply:
I knew I had lost something in translation :D -- we'll talk on the standup 👍